src/pages/Account/Venue/VenueMapEdition/DraggableSubvenue.scss

```diff
@@ -0,0 +1,18 @@
+@import "scss/constants";
+
+.DraggableSubvenue {
+  &__title {
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    display: flex;
+    align-content: center;
+    justify-content: center;
+    background-color: opaque-black(0.5);
+    border-radius: $border-radius--xl;
+    padding: $spacing--xs $spacing--sm;
+    font-size: $font-size--sm;
+    transition: all 400ms $transition-function;
```

Comment:
[OOS] we might need to check all the project's transitions, see if there are few timings we can standardize around and put them in constants similar to `$transition-function` or maybe even as part of.